src/ml/train/focus_classifiers.py

```diff
@@ -0,0 +1,89 @@
+import glob
+import statistics
+import os
+import numpy as np
+
+from sklearn import metrics
+from sklearn.linear_model import LogisticRegression
+from sklearn.dummy import DummyClassifier
+from sklearn.model_selection import train_test_split, cross_val_score
+
+import brainflow
+from brainflow.board_shim import BoardShim, BrainFlowInputParams, LogLevels, BoardIds
+from brainflow.data_filter import DataFilter, FilterTypes, AggOperations, WindowFunctions, DetrendOperations
+from brainflow.ml_model import BrainFlowMetrics, BrainFlowClassifiers, MLModel
+
+
+def prepare_data ():
```

Comment:
I'd say the same about this function. It's too difficult to tell what's going on. 

I see a nested try/except, so i'm sure this could be broken up into logical chunks